src/Microsoft.Identity.Client/Platforms/net45/NavigateErrorStatus.cs

```diff
@@ -104,7 +104,7 @@ internal class NavigateErrorStatus
             },
             {
                 (int)NavigateErrorStatusCode.HTTP_STATUS_NOT_FOUND,
-                "The server has not found anything matching the requested URI (Uniform Scope Identifier)."
+                "The server has not found anything matching the requested URI (Uniform ScopeSet Identifier)."
```

Comment:
it should be scope. We don't have resources anymore. but this file a victim of rename. I will revert it